wazo_webhookd/services/mobile/plugin.py

```diff
@@ -305,9 +305,13 @@ def _send_via_apn(self, message_title, message_body, channel_id, data):
         return requests_automatic_detail(response)
 
     def _create_apn_message(self, message_title, message_body, channel_id, data):
+
+        apns_topic_value = self.config['mobile_apns_call_topic']
```

Comment:
It is not clear what is the difference between `apns_topic_value` and `apns_default_topic_value`. Please rename `apns_topic_value` to something like `apns_call_topic`.
You can also drop the `_value` in variable names, I don't think they bring anything.